recipes/vorpy/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "vorpy" %}
+{% set version = "3.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8529dabbc2a8e3576e8e35cdb8b68f23923d0482f5a0ef75a204cdd23c4feb3d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt"
+  noarch: python
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+    - wheel
+  run:
+    - python >={{ python_min }}
+    - numpy
+    - scipy
+    - matplotlib-base
+    - pandas
+    - plotly
+    - shapely
+    - sympy
+    - numba
+
+test:
+  imports:
+    - vorpy
+  commands:
+    - python -c "import vorpy; print(vorpy.__version__)"
```

Comment:
This is redundant with the `imports` test. Please add a `pip check` though:

https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L64-L67

and also add `python {{ python_min }}` under `requires` as is shown in:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python